bot/exts/evergreen/conversationstarters.py

```diff
@@ -1,5 +1,3 @@
-import json
-import random
 from pathlib import Path
 
 import yaml
```

Comment:
One last thing - this should be in alphabetical order, like this:
```py
from discord import Color, Embed
from discord.ext import commands
import yaml
```

I'm kind of surprised the linting tests didn't catch this.